DirectEmployers/MyJobs
# [PD 3154] enfore readonly non custom cbos

## What's this PR do?
PR enforces that CBO's that aren't custom will only have read only field and won't be editable

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
Pull Down the repository. Go into CBO. On the list, the custom CBO's should have the options to edit and delete and the non-custom ones should only be able to view. If you click through the CBO's and through the Contacts, Locations, Rules, etc of the non custom CBO's you should only be able to view the information for that CBO. You won't be able to update and save anything

## Any background context you want to provide?
Might need to merge other PR's into this one once they are merged in QC before fully merging this one. I still need to work on the categories and the state select dropdown because of the way they are currently built
